packages/docker.rb

```diff
@@ -0,0 +1,161 @@
+# Adapted from Arch Linux docker PKGBUILD at:
+# https://github.com/archlinux/svntogit-community/raw/packages/docker/trunk/PKGBUILD
+
+require 'package'
+
+class Docker < Package
+  description 'Pack, ship and run any application as a lightweight container'
+  homepage 'https://www.docker.com/'
+  version '23.0.0-rc.1'
```

Comment:
Why not the stable version?

Reply:
This is the new stable series to be released shortly.

Reply:
The older series had issues, so it seemed to make more sense to just rebase everything on this new series...